src/test/kotlin/it/pagopa/wallet/controllers/WalletControllerTest.kt

```diff
@@ -119,15 +119,35 @@ class WalletControllerTest {
             .expectStatus()
             .isCreated
             .expectBody()
-            .json(
-                objectMapper.writeValueAsString(
-                    WalletCreateResponseDto()
-                        .walletId(WALLET_DOMAIN.id.value)
-                        .redirectUrl(
-                            "$webviewPaymentUrl#walletId=${WALLET_DOMAIN.id.value}&useDiagnosticTracing=${WalletTestUtils.CREATE_WALLET_REQUEST.useDiagnosticTracing}&paymentMethodId=${WalletTestUtils.CREATE_WALLET_REQUEST.paymentMethodId}"
-                        )
-                )
-            )
+            .jsonPath("$.walletId")
+            .value<String> { walletId ->
+                // Assert that the walletId is as expected
+                assert(walletId.startsWith(WALLET_DOMAIN.id.value.toString().trim())) {
+                    "walletId is not the expected value"
+                }
+            }
+            .jsonPath("$.redirectUrl")
+            .value<String> { redirectUrl ->
+                // Assert that the redirectUrl starts with the expected base URL
+                assert(redirectUrl.contains("#walletId=${WALLET_DOMAIN.id.value}")) {
```

Comment:
```suggestion
                assertTrue(redirectUrl.contains("#walletId=${WALLET_DOMAIN.id.value}")) {
```

use junit jupiter assertTrue message here to have test failure report being populated correctly

Reply:
I respond only on this comment, 
i'm accepting all of them.